EasyPost/Services/BatchService.cs

```diff
@@ -103,13 +106,161 @@ public async Task<Batch> CreateAndBuy(BetaFeatures.Parameters.Batches.Create par
 
         // TODO: Add GetNextPage function when Batches are sorted newest to oldest.
 
+        /// <summary>
+        ///     Add shipments to this batch.
+        /// </summary>
+        /// <param name="parameters">Update shipment parameters.</param>
+        /// <returns>The updated Batch.</returns>
+        [CrudOperations.Update]
+        public async Task<Batch> AddShipments(string id, Dictionary<string, object> parameters)
+        {
+            return await Request<Batch>(Method.Post, $"batches/{id}/add_shipments", parameters);
+        }
+
         /// <summary>
         ///     Retrieve a Batch from its id.
         /// </summary>
         /// <param name="id">String representing a Batch. Starts with "batch_".</param>
         /// <returns>EasyPost.Batch instance.</returns>
         [CrudOperations.Read]
-        public async Task<Batch> Retrieve(string id) => await Get<Batch>($"batches/{id}");
+        public async Task<Batch> Retrieve(string id) => await Request<Batch>(Method.Get, $"batches/{id}");
+
+        /// <summary>
+        ///     Add <see cref="Shipment"/>s to this <see cref="Batch"/>.
+        /// </summary>
+        /// <param name="parameters"><see cref="BetaFeatures.Parameters.Batches.AddShipments"/> parameter set.</param>
+        /// <returns>This updated <see cref="Batch"/> instance.</returns>
+        [CrudOperations.Update]
+        public async Task<Batch> AddShipments(string id, BetaFeatures.Parameters.Batches.AddShipments parameters)
```

Comment:
Issue: need to fix docstring. Please double check on the functions that were copied over, many of them are missing docstring for `id`